connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -190,6 +190,22 @@ class SparkSession(
     range(start, end, step, Option(numPartitions))
   }
 
+  def newSession(): SparkSession = {
+    throw new UnsupportedOperationException("newSession is not supported")
+  }
+
+  def getActiveSession: Option[SparkSession] = {
```

Comment:
These three a part of the object and not the class itself right?

Reply:
right. updated